integration-cli/docker_cli_pull_local_test.go

```diff
@@ -2,10 +2,9 @@ package main
 
 import (
 	"fmt"
+	"github.com/go-check/check"
```

Comment:
This import was in the right place :stuck_out_tongue:. We tend to separate the built-in import to the other ones :wink:.